iXBRLViewerPlugin/viewer/src/js/fact.js

```diff
@@ -191,38 +192,23 @@ Fact.prototype.isNil = function() {
 
 Fact.prototype.readableAccuracy = function () {
     if (!this.isNumeric() || this.isNil()) {
-        return "n/a";
+        return i18next.t("common.notAvailable");
```

Comment:
This is obviously fine, but I would have expected `notApplicable`.

Reply:
Fixed in a4bf05a